packages/icondusk-e2e/package.py

```diff
@@ -24,7 +24,8 @@ class IconduskE2e(CMakePackage):
     depends_on('netcdf-fortran')
     depends_on('netcdf-c')
     depends_on('atlas_utilities', type=('build', 'run'))
-    depends_on('dawn4py',  type=('build', 'run'))
+    depends_on('dawn4py',  type=('build'))
+    depends_on('dusk',  type=('build'))
```

Comment:
Interesting. So basically `icondusk-e2e` will translate the dusk stencils at build time and then link the C++ output into the test executable?

Reply:
that's correct